snapcraft/tests/commands/test_collaborate.py

```diff
@@ -90,3 +91,17 @@ def test_collaborate_bad_request(self):
         self.assertEqual(
                 'Received error 400: "The given `snap-id` does not match '
                 'the assertion\'s."', str(err))
+
+    def test_collaborate_yes_revoke_uploads_request(self):
+        patcher = mock.patch('builtins.input')
+        mock_input = patcher.start()
+        mock_input.return_value = 'y'
+        self.client.login('dummy', 'test correct password')
+        err = self.assertRaises(
+            storeapi.errors.StoreValidationError,
+            collaborate,
+            'revoked', 'keyname')
+
+        self.assertEqual(
+            'Received error 409: "The assertion\'s `developers` would revoke '
```

Comment:
This error message was a nested one, with both `"` and `'`  inside the string. Used str.format() instead now to avoid `\`.